internal/archive/archive.go

```diff
@@ -363,3 +439,13 @@ func sectionPackageInfo(section control.Section) *PackageInfo {
 		SHA256:  section.Get("SHA256"),
 	}
 }
+
+// proSuffixedLabel adds "<pro value> (pro)" suffix to the label and returns it
+// if the archive is specified with pro value. Otherwise, it returns the
+// original label.
+func (index *ubuntuIndex) proSuffixedLabel() string {
```

Comment:
This doesn't look like the archive label.  What is it?

Reply:
This is more like a `displayName`, "label" is a concept that is already well defined in the code. This is a string for presentation purposes we dont want the rest of the code to rely on the implementation, hence remove the contract in the comment.

Reply:
Done in https://github.com/canonical/chisel/pull/167/commits/d1986468ced3757e8f48a616aed5ce2fdc854886.